### ISSUE TITLE
Pyautogui.size() not returning twice of actual screen size.

### ISSUE THREAD

**Issue by ggcandy** [asweigart/pyautogui#436]
Hello. Im on MacOs Catalina. My graphics is Radeon Pro. Ive heard that because of the displays of Macbooks, it causes pyautogui.size() to return twice of the actual value. But problem is, when i do pyautogui.size() it gives me this: `Size(width=1680, height=1050)`. My actual screen size is this: 2880x1800. I have only 1 monitor. How do i fix it?

**Comment by TravisWheeler130** [comment]
Hi there, I am on MacOS Mojave but I am also having the same issue. My computer's resolution is 2880x1800 but pyautogui.size() returns:
Size(width=1440, height=900).
I thought about multiplying everything by 2, but that doesn't seem very proficient.
Thanks for the help!

**Comment by ggcandy** [comment]
> Hi there, I am on MacOS Mojave but I am also having the same issue. My computer's resolution is 2880x1800 but pyautogui.size() returns:
> Size(width=1440, height=900).
> I thought about multiplying everything by 2, but that doesn't seem very proficient.
> Thanks for the help!
> 
> EDIT: I guess maybe not the same issue, but has anyone figured out how to get pyautogui to match the computer's resolution?

yeah i thought it multiplied everything by two, but if it worked that way, then ill happily use it. Problem is it multiplies by 1.714285714285714285714285714285714285714285714285714285....

**Comment by ggcandy** [comment]
Also, I tried re-installing pyautogui and also restarting. Doesnt work =(

**Comment by tatumsu** [comment]
same issue here:
>>> pyautogui.size()
Size(width=1440, height=900)

real screen size is 
![image](https://user-images.githubusercontent.com/2714971/204118717-f441328d-2236-4171-8aa0-4945dafc278d.png)
![image](https://user-images.githubusercontent.com/2714971/204118740-0166e818-b4af-4251-801a-184d6b4e42ac.png)

Python 3.9.6 (default, Sep 26 2022, 11:37:49) 
[Clang 14.0.0 (clang-1400.0.29.202)] on darwin

PyAutoGUI                                         0.9.53
